tests/appsec/iast/utils.py

```diff
@@ -16,18 +16,18 @@ def _get_expectation(d):
     raise TypeError(f"Unsupported expectation type: {d}")
 
 
-def _get_span_meta(request):
+def _get_span_meta(request, metastruct=False):
```

Comment:
`metastruct` is not used ?

Reply:
It was in the end not used anymore. I removed this argument in the latest commit. 😄